src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderResponseContainer.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+
+using SwedbankPay.Sdk.Exceptions;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    public class PaymentOrderResponseContainer
+    { 
+        public PaymentOrderResponseContainer(PaymentOrderResponse paymentOrder)
+        {
+            PaymentOrder = paymentOrder;
+        }
+
+
+        public OperationList Operations { get; set; } = new OperationList();
+
+        public PaymentOrderResponse PaymentOrder { get; set; }
+
+
+        public Uri GetPaymentUrl()
```

Comment:
The other methods are quite similar to this, containing a lot of boilerplate code. Consider finding a way to merge them somehow.